frontend/src/components/pages/Canvas/Canvas.js

```diff
@@ -0,0 +1,114 @@
+
+// var GamePiece;
+
+var Marker;
+
+var Background;
+
+document.addEventListener("DOMContentLoaded", function(event) {
+
+
+    function startGame() {
+        Background = new component(window.innerWidth, window.innerHeight, "/pictures/images-for-canvas/map/map.jpg", 0, 0, "image");
+
+        Marker = new component(125, 200, "/pictures/images-for-canvas/marker/marker.svg", 780, 350, "image");
+
+        // GamePiece = new component(10, 20, "images/cursor/cursor_default.jpg", 0,0, "image");
+        GameArea.start();
+    }
+    startGame();
+})
+console.log(window.location.pathname)
+
+
+var GameArea = {
+    canvas : document.createElement("canvas"),
+    start : function() {
+        this.canvas.width = window.innerWidth;
+        this.canvas.height = window.innerHeight;
+        // this.canvas.style.cursor = "none";
```

Comment:
Commented code is not good to have in the production code 😃

Reply:
ok